src/Doctrine/PlatenumDoctrineType.php

```diff
@@ -101,19 +101,30 @@ public function getName(): string
         return $this->platenumAlias;
     }
 
-    public function getSQLDeclaration(array $declaration, AbstractPlatform $platform): string
+    /**
+     * @psalm-suppress ParamNameMismatch
```

Comment:
please one-line this docblock

Reply:
I am used to add one-line docblocks above properties or statements, for classes/methods I always use multiline. But I changed it as requested.